crates/websocket/src/chat_server.rs

```diff
@@ -288,14 +288,16 @@ impl ChatServer {
     &self,
     op: &OP,
     response: &Response,
-    community_id: CommunityId,
+    community_id: Option<CommunityId>,
     websocket_id: Option<ConnectionId>,
   ) -> Result<(), LemmyError>
   where
     OP: OperationType + ToString,
     Response: Serialize,
   {
     let res_str = &serialize_websocket_message(op, response)?;
+    // use admin room if no community is specified
+    let community_id = community_id.unwrap_or(CommunityId(0));
```

Comment:
Either we have a `None` meaning all communities, or CommunityId(0), but not both.

Reply:
I suppose we could add a separete admin_room, but I dont have the time to test that properly now. Gonna revert the change with optional community id.